### DISCUSSION TITLE
Can't login into my admin account

### DISCUSSION THREAD

**Discussion by kubanedo** [supabase/supabase#30687] [General]
Hi all,

I am trying to log into my account on supabase.com. I forgot my password but resetting password doesn't work. I got redirected to sign in page. The url before redirection says link has expired, but I clicked on the link in the email immediately after it came.

Any ideas what to do?

<img width="792" alt="Snímek obrazovky 2024-11-27 v 11 22 01" src="https://github.com/user-attachments/assets/eff806ab-46fd-468a-9748-16e9037bacbb">

Thanks

**Comment by silentworks** [comment]
You probably signed up through GitHub and not email/password.

**Reply by kubanedo** [reply]
Yes, already figured this out. Thanks :)
